Scenarios/CGCT/CGCT/ViewController.m

```diff
@@ -84,6 +85,7 @@ - (void)viewDidLoad {
     [_demoList addObject:[[DemoScenarioIntro alloc] init]];
     [_demoList addObject:[[DemoScenarioText alloc] init]];
     [_demoList addObject:[[DemoScenarioImages alloc] init]];
```

Comment:
DemoScenarioText is broken despite not being modified. Fails on render in line 580 of CALayer. Requires more investigation.